DigitalLearningSolutions.Data/Extensions/ConfigurationExtensions.cs

```diff
@@ -48,6 +48,8 @@ public static class ConfigurationExtensions
         private const string LearningHubAuthenticationClientSecret = "LearningHubAuthentication:ClientSecret";
 
         private const string LearningHubUserAPIUserAPIUrl = "LearningHubUserApi:UserApiUrl";
+        private const string UserResearchUrlName = "UserResearchUrl";
+
         public static string GetAppRootPath(this IConfiguration config)
         {
             return config[AppRootPathName];

```

Comment:
Can we address the GithubActions warnings appearing here?

Reply:
ok